snuba/cli/devserver.py

```diff
@@ -6,15 +6,19 @@
 @click.option('--workers/--no-workers', default=True)
 def devserver(bootstrap, workers):
     "Starts all Snuba processes for local development."
+    import logging
     import os
     import sys
     from subprocess import list2cmdline, call
     from honcho.manager import Manager
 
+    log_level = logging.getLevelName(logging.getLogger().getEffectiveLevel())
+    command = ['snuba', f'--log-level={log_level}']
```

Comment:
This is pretty clunky and only really works because we use `logging.basicConfig` (which is also pretty clunky.) It'd be better to handle this with [context objects](https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts) but I ended up spending more time making this change than I expected to anyway.